src/main/java/org/folio/marc4ld/service/marc2ld/mapper/mapper/OtherVersionMapper.java

```diff
@@ -0,0 +1,134 @@
+package org.folio.marc4ld.service.marc2ld.mapper.mapper;
+
+import static org.folio.ld.dictionary.PredicateDictionary.INSTANTIATES;
+import static org.folio.ld.dictionary.PredicateDictionary.MAP;
+import static org.folio.ld.dictionary.PredicateDictionary.OTHER_VERSION;
+import static org.folio.ld.dictionary.PredicateDictionary.TITLE;
+import static org.folio.ld.dictionary.PropertyDictionary.LABEL;
+import static org.folio.ld.dictionary.PropertyDictionary.MAIN_TITLE;
+import static org.folio.ld.dictionary.PropertyDictionary.NAME;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.IDENTIFIER;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.ID_LCCN;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.ID_UNKNOWN;
+import static org.folio.marc4ld.util.Constants.S;
+import static org.folio.marc4ld.util.Constants.T;
+import static org.folio.marc4ld.util.Constants.TAG_776;
+import static org.folio.marc4ld.util.MarcUtil.getSubfieldValue;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import lombok.RequiredArgsConstructor;
+import org.folio.ld.dictionary.PredicateDictionary;
+import org.folio.ld.dictionary.ResourceTypeDictionary;
+import org.folio.ld.dictionary.model.Resource;
+import org.folio.ld.dictionary.model.ResourceEdge;
+import org.folio.ld.fingerprint.service.FingerprintHashService;
+import org.folio.marc4ld.dto.MarcData;
+import org.folio.marc4ld.service.label.LabelService;
+import org.folio.marc4ld.service.marc2ld.mapper.Marc2ldMapper;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class OtherVersionMapper implements Marc2ldMapper {
+
+  private static final Set<ResourceTypeDictionary> IDENTIFIER_TYPE = Set.of(IDENTIFIER);
+  private static final String DLC_PREFIX = "(DLC)";
+
+  private final MapperHelper mapperHelper;
+  private final LabelService labelService;
+  private final FingerprintHashService hashService;
+
+  @Override
+  public String getTag() {
+    return TAG_776;
+  }
+
+  @Override
+  public boolean canMap(PredicateDictionary predicate) {
+    return predicate == OTHER_VERSION;
+  }
+
+  @Override
+  public void map(MarcData marcData, Resource resource) {
+    var dataField = marcData.getDataField();
+    mapWork(resource, getSubfieldValue(S, dataField));
+    mapInstance(resource, getSubfieldValue(T, dataField));
+    adjustWorkToInstanceRelation(resource);
+  }
+
+  private void mapWork(Resource resource, String title) {
+    var properties = mapperHelper.getProperties(resource);
+    properties.put(LABEL.getValue(), List.of(title));
+    resource.setDoc(mapperHelper.getJsonNode(properties));
+    mapTitle(resource, title);
+    resource.setLabel(title);
+  }
+
+  private void mapTitle(Resource resource, String title) {
```

Comment:
Due to the new `TitlePreprocessor`, the $s and $t will be always present in 776. 
So, we can map the title directly in the yaml configuration. No need to add any logic in the Java class. right?

Ie, add the following to the yaml mappings

Line 1624:
      `s: MAIN_TITLE`


Line 1683:
```
              subfields:
                t: MAIN_TITLE
```

Reply:
Good point. Thank you!

Reply:
Done